bundlewrap/utils/__init__.py

```diff
@@ -74,7 +74,16 @@ class Fault:
     This let's us gracefully skip items that require information that's
     currently not available.
     """
-    def __init__(self, callback, **kwargs):
+    def __init__(self, fault_identifier, callback, **kwargs):
+        if isinstance(fault_identifier, list):
+            self.id_list = fault_identifier
+        else:
+            self.id_list = [fault_identifier]
+
+        for key, value in sorted(kwargs.items()):
+            self.id_list.append(hash(key))
+            self.id_list.append(hash(value))
```

Comment:
`hash()` return values only remain constant within the same process. So this would break if we ever pickled a Fault and imported it into another process. But that is probably not going to happen.